account_receipt_report/README.rst

```diff
@@ -0,0 +1,81 @@
+======================
+Account Receipt Report
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-payment/tree/11.0/account_receipt_report
+    :alt: OCA/account-payment
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-payment-11-0/account-payment-11-0-account_receipt_report
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/96/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to print a payment receipt when you register an invoice
+payment or when you reconcile a bank statement line.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Go to any invoice and register a payment.
+#. In payment wizard you will see a new button **Validate and print receipt**.
+   If you click in this button, a receipt of the payment will be printed and
+   the invoice will be payed.
+#. Or you can go to a bank statement and you can print the report when line
```

Comment:
The receipt is printed from bank statement line, but not when going to be reconciled.